ampd/src/broadcaster/accounts.rs

```diff
@@ -30,13 +30,14 @@ where
 
     let account = response
         .account
-        .ok_or_else(|| Error::AccountNotFound {
-            address: address.clone(),
+        .ok_or_else(|| {
+            Error::AccountNotFound {
+                address: address.clone(),
+            }
+            .into()
         })
-        .into_report()
```

Comment:
One of the other reasons I didn't upgrade was also that I found `.into_report` quite nice haha. Anyway.